CONTRIBUTING.md

```diff
@@ -20,14 +21,19 @@ $ cd cppcache/test/<Debug|Release|if needed>
 $ ./apache-geode_unittests
 ```
 
-### Running old integration tests
+### Running integration tests
+There are two sets of integration tests based on different testing frameworks: CTest and [Google Test](https://github.com/google/googletest).
+
+CTest-based integration tests are deprecated. If your changes include the implementation of new integration test/s to be verified, they should be written using Google Test. If your change implies a significant change in one or more CTest-based test cases, you should create the equivalent test case/s using Google Test to substitute the old one/s instead of adapting the old ones.
+
+#### Running CTest integration tests (deprecated)
 ```bash
 $ cd <clone>
 $ cd build
 $ cmake --build . --target run-integration-tests
```

Comment:
This should actually be running both the old and the new integration tests. If not it's a bug. ;)

Reply:
I have been testing this command. I changed the code to run just one of the 110 old test cases. I was expecting that after that test case, the new ones were executed, but it did not happen:

```
$ cmake --build . --target run-integration-tests
Test project /home/alb3rtobr/git/Nordix/geode-native/build/cppcache/integration-test
    Start 1: testCacheless
1/1 Test #1: testCacheless ....................   Passed   69.02 sec

100% tests passed, 0 tests failed out of 1

Label Time Summary:
STABLE    =  69.02 sec*proc (1 test)

Total Test time (real) =  69.03 sec
[  0%] Built target run-stable-cppcache-integration-tests
[  0%] Built target version-header
[  2%] Built target ACE-extern
[  6%] Built target boost-extern
[ 10%] Built target xerces-c-extern
[ 80%] Built target apache-geode
[ 84%] Built target framework
[ 92%] Built target testobject
[ 92%] Built target test-cppcache-utils
[ 92%] Built target cryptoImpl
[ 94%] Built target DHImpl
[ 98%] Built target sqlite-extern
[ 98%] Built target SqLiteImpl
[ 98%] Built target securityImpl
[100%] Built target testCacheless
Scanning dependencies of target integration-tests
[100%] Built target integration-tests
Scanning dependencies of target run-integration-tests
[100%] Built target run-integration-tests

```